### PR TITLE
[Translation] Merge italian translation again

### PR DESCRIPTION
**Pull request by ann0see**
Reopens #3. There are still some missing translations in the _includes folder.
Also the general.yml file has changed (I added the footer content there to mitigate this problem)

### CODE REVIEW

File: _data/it/general.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+footer:
+  alt:
```
Comment (ann0see) [inline _data/it/general.yml:2]: I'm really sorry ;-(. You now need to add the copyright text here.

In english it's `Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/corrados/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>`

**Review by ann0see — COMMENTED** [review]
All the other things seem to be allright.

**Comment by dzpex** [comment]
I updated the general.yml and created the it file under _includes folder. Try now and tell me if any problem!

**Review by ann0see — COMMENTED** [review]
Thanks

File: _includes/wiki/it/footertext.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Scarica Jamulus](https://sourceforge.net/projects/llcon/files/){:.button}
```
Comment (ann0see) [inline _includes/wiki/it/footertext.md:1]: You need to add a new line and a [ before Scarica. It's something strange in the plugin.

File: _includes/wiki/it/wikilicense.html [hunk 3]
```diff
@@ -0,0 +1 @@
+[La licenza di questo wiki è soggetta a <a href="https://creativecommons.org/licenses/by-sa/4.0/deed" target="_blank" rel="noreferrer">"Creative Commons Attribution-ShareAlike 4.0 International License"</a>.
```
Comment (ann0see) [inline _includes/wiki/it/wikilicense.html:1]: This [ is too much ;-).

Comment (ann0see) [reply _includes/wiki/it/wikilicense.html:1]: I will remove this and try a merge.

**Review by ann0see — COMMENTED** [review]
Let's try it.

**Pull request merged**